sidecar/sokuji_sidecar/accel.py

```
@@ -285,10 +285,32 @@ def resolve_translate(model_id, override="auto", machine=None, reserved_bytes=0,
+    check is native_models.model_status with the repo override — which carries
+    the partial-snapshot/.incomplete guards a bare snapshot_download lacks."""
+    from . import native_models
+    out = set()
+    for d in model.deployments:
+        if d.compute_type in out:
+            continue
+        try:
+            if native_models.model_status(model.id, repo=d.artifact) == "ready":
+                out.add(d.compute_type)
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)